cpp/src/arrow/array/array_nested.h

```diff
@@ -599,17 +599,23 @@ class ARROW_EXPORT FixedSizeListArray : public Array {
   ///
   /// \param[in] values Array containing list values
   /// \param[in] list_size The fixed length of each list
+  /// \param[in] null_bitmap Optional validity bitmap
   /// \return Will have length equal to values.length() / list_size
-  static Result<std::shared_ptr<Array>> FromArrays(const std::shared_ptr<Array>& values,
-                                                   int32_t list_size);
+  static Result<std::shared_ptr<Array>> FromArrays(
+      const std::shared_ptr<Array>& values,
+      int32_t list_size,
+      std::shared_ptr<Buffer> null_bitmap = NULLPTR);
```

Comment:
Other methods that allow passing a null bitmap (such as `ListArray::FromArrays()`) support another parameter `null_count` to pass the computed null count. Could we support that parameter in C++ as well? We don't need to expose it in Python.

https://github.com/apache/arrow/blob/6b11b42fcc6cd1feb7afc03badbb0deca38dfeef/cpp/src/arrow/array/array_nested.h#L174-L178